README.md

```diff
@@ -1,2 +1,85 @@
-# crave-calculator-android
-crave-calculator-android
+
+<h1 align="center">
+  <img src="https://avatars1.githubusercontent.com/u/16109321?s=200&v=4" width="150">
+  <br>
+  Calculator Android Fragment
+</h1>
+
+<h4 align="center">An open source calculator created by <a href="https://cravehq.com/" target="_blank">Crave</a>.</h4>
+
+<p align="center">
+  <a href="#key-features">Key Features</a> •
+  <a href="#how-to-use">How To Use</a> •
+  <a href="#license">License</a>
+</p>
+	
+<p align="center">
+	<img src="https://user-images.githubusercontent.com/17592866/54139204-cefe3b00-43ff-11e9-89f9-56eed399c6f5.gif" width="250">
+</p>
+
+## Key Features
+* Evaluating expressions
+  - The calculator implement a small library <a href="https://www.objecthunter.net/exp4j/index.html">exp4j</a>, that implements <a href="https://en.wikipedia.org/wiki/Shunting-yard_algorithm">Dijkstra's Shunting Yard Algorithm</a>
+* Fragment
+  - Use it easily in Activities and Fragments
+  - Give it a delegate to be called when any calc update happen
+* Kotlin language
+  - Implemented using Kotlin language
+
+## How To Use
+
+Add the <a href="https://jitpack.io/">JitPack</a> in your `build.gradle` file:
+
+```gradle
+allprojects {
+	repositories {
+    ...
+		maven { url 'https://jitpack.io' }
+    ...
+	}
+}
+```
+
+After that add the library in your dependencies file. Check the last <a href="https://github.com/CraveFood/calculator-android/releases">release</a>
+
+```gradle
+dependencies {
+...
+	implementation "com.github.CraveFood:calculator-android:$calculatorAndroidVersion"
```

Comment:
Let's use an actual version here to simplify things, specially since it doesn't demonstrate how to declare the variable: `implementation "com.github.CraveFood:calculator-android:0.0.3"`